scripts/airflow/tasks/conflate_artery_centreline/B1_arteries_intersection_manual_corr.sql

```diff
@@ -0,0 +1,11 @@
+CREATE SCHEMA IF NOT EXISTS counts;
+
+-- TODO: figure out how / whether to handle intersection corrections here
```

Comment:
can we at least take a stab at this?

IIRC main difficulty is that we need to reverse-engineer the intersection ID from the neighbouring segments, which is what @aharpalaniTO 's process in our existing `flow_geocoding` DAG does.  if we can just drop that in, we should do that!